libc/src/__support/big_int.h

```diff
@@ -469,7 +469,7 @@ struct BigInt {
                                                     !cpp::is_same_v<T, bool>>>
   LIBC_INLINE constexpr BigInt(T v) {
     constexpr size_t T_SIZE = sizeof(T) * CHAR_BIT;
-    const bool is_neg = Signed && (v < 0);
+    const bool is_neg = v < 0;
```

Comment:
I should add a test case to libc/test/src/__support/big_int_test.cpp that demonstrates _why_ this change is necessary.

Reply:
https://github.com/llvm/llvm-project/pull/115084/commits/be223620e9c8466ca88e4b4f3cf06ce92cfe329f